hack/make/.integration-daemon-setup

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+bundle .ensure-emptyfs
+bundle .ensure-frozen-images
+bundle .ensure-httpserver
```

Comment:
The original idea was to have an one file to source to start an interactive shell, where one can start and debug single integration tests.